tools/lightning/csv.md

```diff
@@ -192,11 +192,11 @@ trim-last-separator = false
 Example content:
 
 ```
-ID	Region	Count
-1	East	32
-2	South	NULL
-3	West	10
-4	North	39
+ID    Region    Count
```

Comment:
These tabs here are intentional though.

Reply:
@kennytm Tabs in our documents cause CI failure, so we should replace them with blanks.

Reply:
But it is not showing "tab" separated values format then.